config/configload/configload.go

```diff
@@ -0,0 +1,64 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package configload implements the configuration Loader struct.
+package configload
```

Comment:
Should we have this in `config` package directly? Or this will cause circular dependencies and we cannot have it at this point, if that is the case we can file an issue to investigate if this will be a possibility.

Reply:
Forget about this it will not be possible

Reply:
I don't think it's currently possible, for example `config` imports `testcomponents`, which imports `exporterhelper`, which imports `configload`. Maybe after some major refactor it would be possible but I don't think it's worth it.